troposphere/static/js/components/modals/instance/InstanceLaunchWizardModal.jsx

```diff
@@ -94,9 +99,10 @@ export default React.createClass({
         // Check if the user has any projects, if not then set view to "PROJECT_VIEW"
         // to create a new one
         let projectList = stores.ProjectStore.getAll();
-        if (projectList) {
-            if (view != "IMAGE_VIEW" && projectList.length === 0) {
-                this.viewProject();
+        let sshKey = stores.SSHKeyStore.getAll();
+        if (sshKey) {
+            if (view != "IMAGE_VIEW" && sshKey.length === 0) {
```

Comment:
I see this let/if statement repeated a lot, I wonder if it can be streamlined into a single function?